demos/link-prediction/metapath2vec-link-prediction.ipynb

```diff
@@ -0,0 +1,969 @@
+{
```

Comment:
<blockquote>... on a heterogeneous graph (with multiple node types) ... how to traverse the different types</blockquote><p> </p><p>Potentially "on a heterogeneous graph (with multiple node types) ... how to traverse the different node types" to be slightly more specific?</p><p> </p><p>Also, I don't think the "References" <code>&lt;a ...&gt;&lt;/a&gt;</code> thing quite works right, e.g. the <code>#refs</code> links on RtD don't jump to it, and the rendering is a bit off (the References text ends up being nested inside a <code>&lt;blockquote&gt;</code> ): <a href="https://stellargraph--1614.org.readthedocs.build/en/1614/demos/link-prediction/metapath2vec-link-prediction.html" target="_blank">https://stellargraph--1614.org.readthedocs.build/en/1614/demos/link-prediction/metapath2vec-link-prediction.html</a> (#1502 and #1508 and specifically <a href="https://github.com/stellargraph/stellargraph/pull/1508/commits/9faae4adbafbffa65183c116450f715bcd638686" target="_blank">https://github.com/stellargraph/stellargraph/pull/1508/commits/9faae4adbafbffa65183c116450f715bcd638686</a> are relevant)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1614/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>